lib/matplotlib/texmanager.py

```diff
@@ -63,7 +62,7 @@ class TexManager:
     Repeated calls to this constructor always return the same instance.
     """
 
-    _texcache = os.path.join(mpl.get_cachedir(), 'tex.cache')
+    _texcache = Path(mpl.get_cachedir(), 'tex.cache')
```

Comment:
Maybe rename to `_cache_dir` or `_cache_path`.   The `tex` part is not important here/evident from the context of the class. But that `_texcache` is a Path is not obvious

Reply:
sure